.gitignore

```diff
@@ -23,6 +23,13 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+<<<<<<< HEAD
```

Comment:
This is a remnant from a merge conflict. You should probably remove this from your gitignore.